packages/dd-trace/test/opentracing/span.spec.js

```diff
@@ -163,6 +164,54 @@ describe('Span', () => {
     expect(span.context()._trace.tags['_dd.p.tid']).to.match(/^[a-f0-9]{8}0{8}$/)
   })
 
+  describe('apmTracingEnabled', () => {
+    it('should not add _dd.apm.enabled tag when apmTracingEnabled = undefined', () => {
+      span = new Span(tracer, processor, prioritySampler, {
+        operationName: 'operation',
+        apmTracingEnabled: true
```

Comment:
Should it be `undefined` as per the test title `when apmTracingEnabled = undefined` , right?